src/components/PassportPage.js

```diff
@@ -1,17 +1,62 @@
 import React from 'react';
-import axios from 'axios';
+import { getUserData } from '../actions/loginActions';
 
 class PassportPage extends React.Component {
+  state = {
+    username: '',
+    avatarUrl: '',
+    githubUrl: ''
+  }
+  
   componentDidMount() {
-    axios.get('/api/user').then(res => {
-      console.log(res);
-      console.log(res.data);
-    });
+    getUserData().then(
+      (res) => {
+        const { avatarUrl, githubUrl, username } = res;
+        this.setState({ username, avatarUrl, githubUrl });
+      },
+      (err) => console.log // do something with error? display in UI?
+    );
+  }
+  
+  hanleChange = (e) => {
```

Comment:
@bonham000 good catch on the typo, and yeah, I forgot to add the event handler to the actual input, saw that you fixed this though. Good call, I did this all very quickly before bed last night 🙁